spiketoolkit/preprocessing/suppress_artifacts.py

```diff
@@ -0,0 +1,118 @@
+from scipy.signal import convolve
+from spikeextractors import RecordingExtractor
+from spikeextractors.extraction_tools import check_get_traces_args
+
+from spiketoolkit.preprocessing.basepreprocessorrecording import (
+    BasePreprocessorRecordingExtractor,
+)
+import numpy as np
+
+
+class ArtifactSuppressionRecording(BasePreprocessorRecordingExtractor):
+    preprocessor_name = "Clip"
+    installed = True  # check at class level if installed or not
+    installation_mesg = ""  # err
+
+    def __init__(
+        self,
+        recording,
+        thresh=2000,
+        ms_surrounding=50,
+        fill_mode="zero",
+        noise_fill_std=3,
+    ):
+        if not isinstance(recording, RecordingExtractor):
+            raise ValueError("'recording' must be a RecordingExtractor")
+        self._thresh = thresh
+        self._ms_surrounding = ms_surrounding
+        self._fill_mode = fill_mode
+        self._noise_fill_std = noise_fill_std
+        BasePreprocessorRecordingExtractor.__init__(self, recording)
+        self.has_unscaled = False
```

Comment:
```suggestion
```

Already taken car of by base (https://github.com/SpikeInterface/spiketoolkit/blob/master/spiketoolkit/preprocessing/basepreprocessorrecording.py#L22-L25)